src/integrationTest/sam.test.ts

```diff
@@ -404,6 +413,12 @@ describe('SAM Integration Tests', async function () {
                         testConfig.lambda = {
                             runtime: scenario.runtime,
                         }
+
+                        // little hack for Go, have to set GOPROXY to direct or it will fail to build
```

Comment:
is there a doc or post that hopefully has more detail about this? Then please include the URL here.